### PR TITLE
Lint quicksilver proto files

### PR DESCRIPTION
**Pull request by pysel**
## 1. Summary
Fixes # (issue)

## 2.Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## 5. Checklist

- [ ] Does the Readme need to be updated?

### CODE REVIEW

**Review by tropicaldog — CHANGES_REQUESTED** [review]
after setup the Buf version to v1.21.0 and run `make proto-lint`, I got this: 

```
proto/quicksilver/airdrop/v1/airdrop.proto:16:3:Enum value name "ActionUndefined" should be prefixed with "ACTION_".
proto/quicksilver/airdrop/v1/airdrop.proto:16:3:Enum value name "ActionUndefined" should be UPPER_SNAKE_CASE, such as "ACTION_UNDEFINED".
proto/quicksilver/airdrop/v1/airdrop.proto:16:3:Enum zero value name "ActionUndefined" should be suffixed with "_UNSPECIFIED".
proto/quicksilver/airdrop/v1/airdrop.proto:18:3:Enum value name "ActionInitialClaim" should be prefixed with "ACTION_".
proto/quicksilver/airdrop/v1/airdrop.proto:18:3:Enum value name "ActionInitialClaim" should be UPPER_SNAKE_CASE, such as "ACTION_INITIAL_CLAIM".
proto/quicksilver/airdrop/v1/airdrop.proto:20:3:Enum value name "ActionDepositT1" should be prefixed with "ACTION_".
proto/quicksilver/airdrop/v1/airdrop.proto:20:3:Enum value name "ActionDepositT1" should be UPPER_SNAKE_CASE, such as "ACTION_DEPOSIT_T1".
proto/quicksilver/airdrop/v1/airdrop.proto:22:3:Enum value name "ActionDepositT2" should be prefixed with "ACTION_".
proto/quicksilver/airdrop/v1/airdrop.proto:22:3:Enum value name "ActionDepositT2" should be UPPER_SNAKE_CASE, such as "ACTION_DEPOSIT_T2".
proto/quicksilver/airdrop/v1/airdrop.proto:24:3:Enum value name "ActionDepositT3" should be prefixed with "ACTION_".
proto/quicksilver/airdrop/v1/airdrop.proto:24:3:Enum value name "ActionDepositT3" should be UPPER_SNAKE_CASE, such as "ACTION_DEPOSIT_T3".
proto/quicksilver/airdrop/v1/airdrop.proto:26:3:Enum value name "ActionDepositT4" should be prefixed with "ACTION_".
proto/quicksilver/airdrop/v1/airdrop.proto:26:3:Enum value name "ActionDepositT4" should be UPPER_SNAKE_CASE, such as "ACTION_DEPOSIT_T4".
proto/quicksilver/airdrop/v1/airdrop.proto:28:3:Enum value name "ActionDepositT5" should be prefixed with "ACTION_".
proto/quicksilver/airdrop/v1/airdrop.proto:28:3:Enum value name "ActionDepositT5" should be UPPER_SNAKE_CASE, such as "ACTION_DEPOSIT_T5".
proto/quicksilver/airdrop/v1/airdrop.proto:30:3:Enum value name "ActionStakeQCK" should be prefixed with "ACTION_".
proto/quicksilver/airdrop/v1/airdrop.proto:30:3:Enum value name "ActionStakeQCK" should be UPPER_SNAKE_CASE, such as "ACTION_STAKE_QCK".
proto/quicksilver/airdrop/v1/airdrop.proto:32:3:Enum value name "ActionSignalIntent" should be prefixed with "ACTION_".
proto/quicksilver/airdrop/v1/airdrop.proto:32:3:Enum value name "ActionSignalIntent" should be UPPER_SNAKE_CASE, such as "ACTION_SIGNAL_INTENT".
proto/quicksilver/airdrop/v1/airdrop.proto:34:3:Enum value name "ActionQSGov" should be prefixed with "ACTION_".
proto/quicksilver/airdrop/v1/airdrop.proto:34:3:Enum value name "ActionQSGov" should be UPPER_SNAKE_CASE, such as "ACTION_QS_GOV".
proto/quicksilver/airdrop/v1/airdrop.proto:36:3:Enum value name "ActionGbP" should be prefixed with "ACTION_".
proto/quicksilver/airdrop/v1/airdrop.proto:36:3:Enum value name "ActionGbP" should be UPPER_SNAKE_CASE, such as "ACTION_GB_P".
proto/quicksilver/airdrop/v1/airdrop.proto:38:3:Enum value name "ActionOsmosis" should be prefixed with "ACTION_".
proto/quicksilver/airdrop/v1/airdrop.proto:38:3:Enum value name "ActionOsmosis" should be UPPER_SNAKE_CASE, such as "ACTION_OSMOSIS".
proto/quicksilver/airdrop/v1/airdrop.proto:45:3:Enum value name "StatusUndefined" should be prefixed with "STATUS_".
proto/quicksilver/airdrop/v1/airdrop.proto:45:3:Enum value name "StatusUndefined" should be UPPER_SNAKE_CASE, such as "STATUS_UNDEFINED".
proto/quicksilver/airdrop/v1/airdrop.proto:45:3:Enum zero value name "StatusUndefined" should be suffixed with "_UNSPECIFIED".
proto/quicksilver/airdrop/v1/airdrop.proto:46:3:Enum value name "StatusActive" should be prefixed with "STATUS_".
proto/quicksilver/airdrop/v1/airdrop.proto:46:3:Enum value name "StatusActive" should be UPPER_SNAKE_CASE, such as "STATUS_ACTIVE".
proto/quicksilver/airdrop/v1/airdrop.proto:47:3:Enum value name "StatusFuture" should be prefixed with "STATUS_".
proto/quicksilver/airdrop/v1/airdrop.proto:47:3:Enum value name "StatusFuture" should be UPPER_SNAKE_CASE, such as "STATUS_FUTURE".
proto/quicksilver/airdrop/v1/airdrop.proto:48:3:Enum value name "StatusExpired" should be prefixed with "STATUS_".
proto/quicksilver/airdrop/v1/airdrop.proto:48:3:Enum value name "StatusExpired" should be UPPER_SNAKE_CASE, such as "STATUS_EXPIRED".
proto/quicksilver/airdrop/v1/messages.proto:14:9:Service name "Msg" should be suffixed with "Service".
proto/quicksilver/airdrop/v1/messages.proto:15:13:RPC request type "MsgClaim" should be named "ClaimRequest" or "MsgClaimRequest".
proto/quicksilver/airdrop/v1/messages.proto:22:26:RPC request type "MsgIncentivePoolSpend" should be named "IncentivePoolSpendRequest" or "MsgIncentivePoolSpendRequest".
proto/quicksilver/airdrop/v1/query.proto:14:9:Service name "Query" should be suffixed with "Service".
proto/quicksilver/claimsmanager/v1/claimsmanager.proto:14:3:Enum value name "ClaimTypeUndefined" should be prefixed with "CLAIM_TYPE_".
proto/quicksilver/claimsmanager/v1/claimsmanager.proto:14:3:Enum value name "ClaimTypeUndefined" should be UPPER_SNAKE_CASE, such as "CLAIM_TYPE_UNDEFINED".
proto/quicksilver/claimsmanager/v1/claimsmanager.proto:14:3:Enum zero value name "ClaimTypeUndefined" should be suffixed with "_UNSPECIFIED".
proto/quicksilver/claimsmanager/v1/claimsmanager.proto:15:3:Enum value name "ClaimTypeLiquidToken" should be prefixed with "CLAIM_TYPE_".
proto/quicksilver/claimsmanager/v1/claimsmanager.proto:15:3:Enum value name "ClaimTypeLiquidToken" should be UPPER_SNAKE_CASE, such as "CLAIM_TYPE_LIQUID_TOKEN".
proto/quicksilver/claimsmanager/v1/claimsmanager.proto:16:3:Enum value name "ClaimTypeOsmosisPool" should be prefixed with "CLAIM_TYPE_".
proto/quicksilver/claimsmanager/v1/claimsmanager.proto:16:3:Enum value name "ClaimTypeOsmosisPool" should be UPPER_SNAKE_CASE, such as "CLAIM_TYPE_OSMOSIS_POOL".
proto/quicksilver/claimsmanager/v1/claimsmanager.proto:17:3:Enum value name "ClaimTypeCrescentPool" should be prefixed with "CLAIM_TYPE_".
proto/quicksilver/claimsmanager/v1/claimsmanager.proto:17:3:Enum value name "ClaimTypeCrescentPool" should be UPPER_SNAKE_CASE, such as "CLAIM_TYPE_CRESCENT_POOL".
proto/quicksilver/claimsmanager/v1/claimsmanager.proto:18:3:Enum value name "ClaimTypeSifchainPool" should be prefixed with "CLAIM_TYPE_".
proto/quicksilver/claimsmanager/v1/claimsmanager.proto:18:3:Enum value name "ClaimTypeSifchainPool" should be UPPER_SNAKE_CASE, such as "CLAIM_TYPE_SIFCHAIN_POOL".
proto/quicksilver/claimsmanager/v1/claimsmanager.proto:19:3:Enum value name "ClaimTypeUmeeToken" should be prefixed with "CLAIM_TYPE_".
proto/quicksilver/claimsmanager/v1/claimsmanager.proto:19:3:Enum value name "ClaimTypeUmeeToken" should be UPPER_SNAKE_CASE, such as "CLAIM_TYPE_UMEE_TOKEN".
proto/quicksilver/claimsmanager/v1/claimsmanager.proto:36:10:Field name "_amount" should be lower_snake_case, such as "amount".
proto/quicksilver/claimsmanager/v1/messages.proto:7:9:Service name "Msg" should be suffixed with "Service".
proto/quicksilver/claimsmanager/v1/query.proto:13:9:Service name "Query" should be suffixed with "Service".
proto/quicksilver/claimsmanager/v1/query.proto:15:3:"quicksilver.claimsmanager.v1.QueryClaimsRequest" is used as the request or response type for multiple RPCs.
proto/quicksilver/claimsmanager/v1/query.proto:15:3:"quicksilver.claimsmanager.v1.QueryClaimsResponse" is used as the request or response type for multiple RPCs.
proto/quicksilver/claimsmanager/v1/query.proto:20:3:"quicksilver.claimsmanager.v1.QueryClaimsRequest" is used as the request or response type for multiple RPCs.
proto/quicksilver/claimsmanager/v1/query.proto:20:3:"quicksilver.claimsmanager.v1.QueryClaimsResponse" is used as the request or response type for multiple RPCs.
proto/quicksilver/claimsmanager/v1/query.proto:20:23:RPC request type "QueryClaimsRequest" should be named "LastEpochClaimsRequest" or "QueryLastEpochClaimsRequest".
proto/quicksilver/claimsmanager/v1/query.proto:20:52:RPC response type "QueryClaimsResponse" should be named "LastEpochClaimsResponse" or "QueryLastEpochClaimsResponse".
proto/quicksilver/claimsmanager/v1/query.proto:25:3:"quicksilver.claimsmanager.v1.QueryClaimsRequest" is used as the request or response type for multiple RPCs.
proto/quicksilver/claimsmanager/v1/query.proto:25:3:"quicksilver.claimsmanager.v1.QueryClaimsResponse" is used as the request or response type for multiple RPCs.
proto/quicksilver/claimsmanager/v1/query.proto:25:18:RPC request type "QueryClaimsRequest" should be named "UserClaimsRequest" or "QueryUserClaimsRequest".
proto/quicksilver/claimsmanager/v1/query.proto:25:47:RPC response type "QueryClaimsResponse" should be named "UserClaimsResponse" or "QueryUserClaimsResponse".
proto/quicksilver/claimsmanager/v1/query.proto:30:3:"quicksilver.claimsmanager.v1.QueryClaimsRequest" is used as the request or response type for multiple RPCs.
proto/quicksilver/claimsmanager/v1/query.proto:30:3:"quicksilver.claimsmanager.v1.QueryClaimsResponse" is used as the request or response type for multiple RPCs.
proto/quicksilver/claimsmanager/v1/query.proto:30:27:RPC request type "QueryClaimsRequest" should be named "UserLastEpochClaimsRequest" or "QueryUserLastEpochClaimsRequest".
proto/quicksilver/claimsmanager/v1/query.proto:30:56:RPC response type "QueryClaimsResponse" should be named "UserLastEpochClaimsResponse" or "QueryUserLastEpochClaimsResponse".
proto/quicksilver/interchainstaking/v1/genesis.proto:10:9:Message name "Params_v1" should be PascalCase, such as "ParamsV1".
proto/quicksilver/interchainstaking/v1/interchainstaking.proto:52:15:Field name "subzoneInfo" should be lower_snake_case, such as "subzone_info".
proto/quicksilver/interchainstaking/v1/interchainstaking.proto:62:10:Field name "base_chainID" should be lower_snake_case, such as "base_chain_id".
proto/quicksilver/interchainstaking/v1/interchainstaking.proto:97:10:Field name "_amount" should be lower_snake_case, such as "amount".
proto/quicksilver/interchainstaking/v1/interchainstaking.proto:147:9:Field name "_amount" should be lower_snake_case, such as "amount".
proto/quicksilver/interchainstaking/v1/messages.proto:13:9:Service name "Msg" should be suffixed with "Service".
proto/quicksilver/interchainstaking/v1/messages.proto:16:25:RPC request type "MsgRequestRedemption" should be named "RequestRedemptionRequest" or "MsgRequestRedemptionRequest".
proto/quicksilver/interchainstaking/v1/messages.proto:24:20:RPC request type "MsgSignalIntent" should be named "SignalIntentRequest" or "MsgSignalIntentRequest".
proto/quicksilver/interchainstaking/v1/messages.proto:33:24:RPC request type "MsgCancelQueuedRedemption" should be named "CancelRedemptionRequest" or "MsgCancelRedemptionRequest".
proto/quicksilver/interchainstaking/v1/messages.proto:33:60:RPC response type "MsgCancelQueuedRedemptionResponse" should be named "CancelRedemptionResponse" or "MsgCancelRedemptionResponse".
proto/quicksilver/interchainstaking/v1/messages.proto:42:23:RPC request type "MsgGovCloseChannel" should be named "GovCloseChannelRequest" or "MsgGovCloseChannelRequest".
proto/quicksilver/interchainstaking/v1/messages.proto:49:24:RPC request type "MsgGovReopenChannel" should be named "GovReopenChannelRequest" or "MsgGovReopenChannelRequest".
proto/quicksilver/interchainstaking/v1/messages.proto:56:21:RPC request type "MsgGovSetLsmCaps" should be named "GovSetLsmCapsRequest" or "MsgGovSetLsmCapsRequest".
proto/quicksilver/interchainstaking/v1/messages.proto:63:31:RPC request type "MsgGovAddValidatorDenyList" should be named "GovAddValidatorDenyListRequest" or "MsgGovAddValidatorDenyListRequest".
proto/quicksilver/interchainstaking/v1/messages.proto:70:34:RPC request type "MsgGovRemoveValidatorDenyList" should be named "GovRemoveValidatorDenyListRequest" or "MsgGovRemoveValidatorDenyListRequest".
proto/quicksilver/interchainstaking/v1/query.proto:15:9:Service name "Query" should be suffixed with "Service".
proto/quicksilver/interchainstaking/v1/query.proto:31:22:RPC request type "QueryDepositAccountForChainRequest" should be named "DepositAccountRequest" or "QueryDepositAccountRequest".
proto/quicksilver/interchainstaking/v1/query.proto:31:67:RPC response type "QueryDepositAccountForChainResponse" should be named "DepositAccountResponse" or "QueryDepositAccountResponse".
proto/quicksilver/interchainstaking/v1/query.proto:65:3:"quicksilver.interchainstaking.v1.QueryWithdrawalRecordsRequest" is used as the request or response type for multiple RPCs.
proto/quicksilver/interchainstaking/v1/query.proto:65:3:"quicksilver.interchainstaking.v1.QueryWithdrawalRecordsResponse" is used as the request or response type for multiple RPCs.
proto/quicksilver/interchainstaking/v1/query.proto:65:29:RPC request type "QueryWithdrawalRecordsRequest" should be named "ZoneWithdrawalRecordsRequest" or "QueryZoneWithdrawalRecordsRequest".
proto/quicksilver/interchainstaking/v1/query.proto:65:69:RPC response type "QueryWithdrawalRecordsResponse" should be named "ZoneWithdrawalRecordsResponse" or "QueryZoneWithdrawalRecordsResponse".
proto/quicksilver/interchainstaking/v1/query.proto:70:3:"quicksilver.interchainstaking.v1.QueryWithdrawalRecordsRequest" is used as the request or response type for multiple RPCs.
proto/quicksilver/interchainstaking/v1/query.proto:70:3:"quicksilver.interchainstaking.v1.QueryWithdrawalRecordsResponse" is used as the request or response type for multiple RPCs.
proto/quicksilver/interchainstaking/v1/query.proto:70:33:RPC request type "QueryWithdrawalRecordsRequest" should be named "UserZoneWithdrawalRecordsRequest" or "QueryUserZoneWithdrawalRecordsRequest".
proto/quicksilver/interchainstaking/v1/query.proto:70:73:RPC response type "QueryWithdrawalRecordsResponse" should be named "UserZoneWithdrawalRecordsResponse" or "QueryUserZoneWithdrawalRecordsResponse".
proto/quicksilver/interchainstaking/v1/query.proto:77:3:"quicksilver.interchainstaking.v1.QueryWithdrawalRecordsRequest" is used as the request or response type for multiple RPCs.
proto/quicksilver/interchainstaking/v1/query.proto:77:3:"quicksilver.interchainstaking.v1.QueryWithdrawalRecordsResponse" is used as the request or response type for multiple RPCs.
proto/quicksilver/interchainstaking/v1/query.proto:82:3:"quicksilver.interchainstaking.v1.QueryWithdrawalRecordsResponse" is used as the request or response type for multiple RPCs.
proto/quicksilver/interchainstaking/v1/query.proto:82:73:RPC response type "QueryWithdrawalRecordsResponse" should be named "UserWithdrawalRecordsResponse" or "QueryUserWithdrawalRecordsResponse".
proto/quicksilver/interchainstaking/v1/query.proto:107:3:"quicksilver.interchainstaking.v1.QueryClaimedPercentageRequest" is used as the request or response type for multiple RPCs.
proto/quicksilver/interchainstaking/v1/query.proto:107:3:"quicksilver.interchainstaking.v1.QueryClaimedPercentageResponse" is used as the request or response type for multiple RPCs.
proto/quicksilver/interchainstaking/v1/query.proto:112:3:"quicksilver.interchainstaking.v1.QueryClaimedPercentageRequest" is used as the request or response type for multiple RPCs.
proto/quicksilver/interchainstaking/v1/query.proto:112:3:"quicksilver.interchainstaking.v1.QueryClaimedPercentageResponse" is used as the request or response type for multiple RPCs.
proto/quicksilver/interchainstaking/v1/query.proto:112:36:RPC request type "QueryClaimedPercentageRequest" should be named "ClaimedPercentageByClaimTypeRequest" or "QueryClaimedPercentageByClaimTypeRequest".
proto/quicksilver/interchainstaking/v1/query.proto:112:76:RPC response type "QueryClaimedPercentageResponse" should be named "ClaimedPercentageByClaimTypeResponse" or "QueryClaimedPercentageByClaimTypeResponse".
proto/quicksilver/interchainstaking/v1/query.proto:290:22:Field name "RemoteAddressMap" should be lower_snake_case, such as "remote_address_map".
proto/quicksilver/participationrewards/v1/messages.proto:13:9:Service name "Msg" should be suffixed with "Service".
proto/quicksilver/participationrewards/v1/messages.proto:14:19:RPC request type "MsgSubmitClaim" should be named "SubmitClaimRequest" or "MsgSubmitClaimRequest".
proto/quicksilver/participationrewards/v1/messages.proto:23:29:RPC request type "MsgGovRemoveProtocolData" should be named "GovRemoveProtocolDataRequest" or "MsgGovRemoveProtocolDataRequest".
proto/quicksilver/participationrewards/v1/participationrewards.proto:58:3:Enum value name "ProtocolDataTypeUndefined" should be prefixed with "PROTOCOL_DATA_TYPE_".
proto/quicksilver/participationrewards/v1/participationrewards.proto:58:3:Enum value name "ProtocolDataTypeUndefined" should be UPPER_SNAKE_CASE, such as "PROTOCOL_DATA_TYPE_UNDEFINED".
proto/quicksilver/participationrewards/v1/participationrewards.proto:58:3:Enum zero value name "ProtocolDataTypeUndefined" should be suffixed with "_UNSPECIFIED".
proto/quicksilver/participationrewards/v1/participationrewards.proto:59:3:Enum value name "ProtocolDataTypeConnection" should be prefixed with "PROTOCOL_DATA_TYPE_".
proto/quicksilver/participationrewards/v1/participationrewards.proto:59:3:Enum value name "ProtocolDataTypeConnection" should be UPPER_SNAKE_CASE, such as "PROTOCOL_DATA_TYPE_CONNECTION".
proto/quicksilver/participationrewards/v1/participationrewards.proto:60:3:Enum value name "ProtocolDataTypeOsmosisParams" should be prefixed with "PROTOCOL_DATA_TYPE_".
proto/quicksilver/participationrewards/v1/participationrewards.proto:60:3:Enum value name "ProtocolDataTypeOsmosisParams" should be UPPER_SNAKE_CASE, such as "PROTOCOL_DATA_TYPE_OSMOSIS_PARAMS".
proto/quicksilver/participationrewards/v1/participationrewards.proto:61:3:Enum value name "ProtocolDataTypeLiquidToken" should be prefixed with "PROTOCOL_DATA_TYPE_".
proto/quicksilver/participationrewards/v1/participationrewards.proto:61:3:Enum value name "ProtocolDataTypeLiquidToken" should be UPPER_SNAKE_CASE, such as "PROTOCOL_DATA_TYPE_LIQUID_TOKEN".
proto/quicksilver/participationrewards/v1/participationrewards.proto:62:3:Enum value name "ProtocolDataTypeOsmosisPool" should be prefixed with "PROTOCOL_DATA_TYPE_".
proto/quicksilver/participationrewards/v1/participationrewards.proto:62:3:Enum value name "ProtocolDataTypeOsmosisPool" should be UPPER_SNAKE_CASE, such as "PROTOCOL_DATA_TYPE_OSMOSIS_POOL".
proto/quicksilver/participationrewards/v1/participationrewards.proto:63:3:Enum value name "ProtocolDataTypeCrescentPool" should be prefixed with "PROTOCOL_DATA_TYPE_".
proto/quicksilver/participationrewards/v1/participationrewards.proto:63:3:Enum value name "ProtocolDataTypeCrescentPool" should be UPPER_SNAKE_CASE, such as "PROTOCOL_DATA_TYPE_CRESCENT_POOL".
proto/quicksilver/participationrewards/v1/participationrewards.proto:64:3:Enum value name "ProtocolDataTypeSifchainPool" should be prefixed with "PROTOCOL_DATA_TYPE_".
proto/quicksilver/participationrewards/v1/participationrewards.proto:64:3:Enum value name "ProtocolDataTypeSifchainPool" should be UPPER_SNAKE_CASE, such as "PROTOCOL_DATA_TYPE_SIFCHAIN_POOL".
proto/quicksilver/participationrewards/v1/participationrewards.proto:65:3:Enum value name "ProtocolDataTypeUmeeParams" should be prefixed with "PROTOCOL_DATA_TYPE_".
proto/quicksilver/participationrewards/v1/participationrewards.proto:65:3:Enum value name "ProtocolDataTypeUmeeParams" should be UPPER_SNAKE_CASE, such as "PROTOCOL_DATA_TYPE_UMEE_PARAMS".
proto/quicksilver/participationrewards/v1/participationrewards.proto:66:3:Enum value name "ProtocolDataTypeUmeeReserves" should be prefixed with "PROTOCOL_DATA_TYPE_".
proto/quicksilver/participationrewards/v1/participationrewards.proto:66:3:Enum value name "ProtocolDataTypeUmeeReserves" should be UPPER_SNAKE_CASE, such as "PROTOCOL_DATA_TYPE_UMEE_RESERVES".
proto/quicksilver/participationrewards/v1/participationrewards.proto:67:3:Enum value name "ProtocolDataTypeUmeeInterestScalar" should be prefixed with "PROTOCOL_DATA_TYPE_".
proto/quicksilver/participationrewards/v1/participationrewards.proto:67:3:Enum value name "ProtocolDataTypeUmeeInterestScalar" should be UPPER_SNAKE_CASE, such as "PROTOCOL_DATA_TYPE_UMEE_INTEREST_SCALAR".
proto/quicksilver/participationrewards/v1/participationrewards.proto:68:3:Enum value name "ProtocolDataTypeUmeeTotalBorrows" should be prefixed with "PROTOCOL_DATA_TYPE_".
proto/quicksilver/participationrewards/v1/participationrewards.proto:68:3:Enum value name "ProtocolDataTypeUmeeTotalBorrows" should be UPPER_SNAKE_CASE, such as "PROTOCOL_DATA_TYPE_UMEE_TOTAL_BORROWS".
proto/quicksilver/participationrewards/v1/participationrewards.proto:69:3:Enum value name "ProtocolDataTypeUmeeUTokenSupply" should be prefixed with "PROTOCOL_DATA_TYPE_".
proto/quicksilver/participationrewards/v1/participationrewards.proto:69:3:Enum value name "ProtocolDataTypeUmeeUTokenSupply" should be UPPER_SNAKE_CASE, such as "PROTOCOL_DATA_TYPE_UMEE_U_TOKEN_SUPPLY".
proto/quicksilver/participationrewards/v1/participationrewards.proto:70:3:Enum value name "ProtocolDataTypeUmeeLeverageModuleBalance" should be prefixed with "PROTOCOL_DATA_TYPE_".
proto/quicksilver/participationrewards/v1/participationrewards.proto:70:3:Enum value name "ProtocolDataTypeUmeeLeverageModuleBalance" should be UPPER_SNAKE_CASE, such as "PROTOCOL_DATA_TYPE_UMEE_LEVERAGE_MODULE_BALANCE".
proto/quicksilver/participationrewards/v1/participationrewards.proto:71:3:Enum value name "ProtocolDataTypeCrescentParams" should be prefixed with "PROTOCOL_DATA_TYPE_".
proto/quicksilver/participationrewards/v1/participationrewards.proto:71:3:Enum value name "ProtocolDataTypeCrescentParams" should be UPPER_SNAKE_CASE, such as "PROTOCOL_DATA_TYPE_CRESCENT_PARAMS".
proto/quicksilver/participationrewards/v1/participationrewards.proto:72:3:Enum value name "ProtocolDataTypeCrescentReserveAddressBalance" should be prefixed with "PROTOCOL_DATA_TYPE_".
proto/quicksilver/participationrewards/v1/participationrewards.proto:72:3:Enum value name "ProtocolDataTypeCrescentReserveAddressBalance" should be UPPER_SNAKE_CASE, such as "PROTOCOL_DATA_TYPE_CRESCENT_RESERVE_ADDRESS_BALANCE".
proto/quicksilver/participationrewards/v1/participationrewards.proto:73:3:Enum value name "ProtocolDataTypeCrescentPoolCoinSupply" should be prefixed with "PROTOCOL_DATA_TYPE_".
proto/quicksilver/participationrewards/v1/participationrewards.proto:73:3:Enum value name "ProtocolDataTypeCrescentPoolCoinSupply" should be UPPER_SNAKE_CASE, such as "PROTOCOL_DATA_TYPE_CRESCENT_POOL_COIN_SUPPLY".
proto/quicksilver/participationrewards/v1/participationrewards.proto:74:3:Enum value name "ProtocolDataTypeOsmosisCLPool" should be prefixed with "PROTOCOL_DATA_TYPE_".
proto/quicksilver/participationrewards/v1/participationrewards.proto:74:3:Enum value name "ProtocolDataTypeOsmosisCLPool" should be UPPER_SNAKE_CASE, such as "PROTOCOL_DATA_TYPE_OSMOSIS_CL_POOL".
proto/quicksilver/participationrewards/v1/query.proto:11:9:Service name "Query" should be suffixed with "Service".
make: *** [proto-lint] Error 100
```

This PR has addressed some lint issues, but further work is needed to fully resolve all linting errors and warnings on quicksilver's proto.

**Comment by pysel** [comment]
> after setup the Buf version to v1.21.0 and run `make proto-lint`, I got this:
> 
> Full linting output for Quicksilver's proto
> This PR has addressed some lint issues, but further work is needed to fully resolve all linting errors and warnings on quicksilver's proto.

yeah, there are many linting issues, I only fixed those that have popped up in my editor

**Comment by faddat** [comment]
Can you update docs like the rabbit mentioned?

thanks :0

**Comment by pysel** [comment]
> Can you update docs like the rabbit mentioned?
> 
> thanks :0

hey @faddat, done

**Comment by joe-bowman** [comment]
It's not just docs that will be affected by this. The next run of `make proto-gen` will come with a whole bunch of changes to the code base.

Why the dropping of Msg and Query prefixes? These still seem to be common usage until at least cosmos-sdk v0.50.

**Comment by faddat** [comment]
Personally my approach to linter-suggested naming changes is to slavishly accept them.  My reasoning for that is that when we slavishly accept these changes, we get a fixed standard for future use.

@pysel -- for this PR to work though, we would need to regenerate the .proto files and update the docs.

**Comment by faddat** [comment]
@pysel we've still got some tests failing -- and I'll let @joe-bowman be the ultimate judge of weather or not to merge this, but .... yeah my stance is to keep bringing them in.

File: proto/quicksilver/interchainquery/v1/query.proto [hunk 1]
```diff
@@ -13,21 +13,21 @@ import "tendermint/types/types.proto";
...
 // QueryParamsRequest is the request type for the Query/Params RPC method.
-message QueryRequestsRequest {
+message QueriesRequest {
   cosmos.base.query.v1beta1.PageRequest pagination = 1;
   string chain_id = 2;
 }
 
 // QueryParamsResponse is the response type for the Query/Params RPC method.
-message QueryRequestsResponse {
+message QueriesResponse {
```
[the remark this answers is not in the document]
Comment (faddat) [reply proto/quicksilver/interchainquery/v1/query.proto]: @pysel could you check this one out please?

**Comment by joe-bowman** [comment]
This pull request has been automatically marked as stale because it has not had any recent activity. It will be closed if no further activity occurs. Thank you!

**Comment by joe-bowman** [comment]
This pull request has been automatically marked as stale because it has not had any recent activity. It will be closed if no further activity occurs. Thank you!

**Pull request closed without merging**